docs/skeleton_annotation.md

```diff
@@ -98,9 +109,9 @@ Hover over existing tree groups to bring up a little menu for creating new group
 Common tree operations include splitting and merging trees.
 
 - `Tree splitting` can be done in two ways:
-  1. Select a node at which to split (*SHIFT + Left Click*) and delete (*DEL*) it.
-  2. Delete an edge between two nodes. Select the first node (*SHIFT + Left Click*), then *SHIFT + CTRL + Left Click* on the second node of the edge to delete this connection.
-- `Tree merging` works similarly to edge deletion but will create a new edge between two previously unconnected trees. Select the first node and *SHIFT + ALT + Left Click* the second node to create an edge between the two.
+  1. Delete a node at which to split. This can be done by rightclicking a node and choosing "Delete this Node". If you have enabled *Classic Controls*, you need to select (*SHIFT + Left Click*) the node first and then delete (*DEL*) it.
```

Comment:
```suggestion
  1. Delete a node at which to split. This can be done by right-clicking a node and choosing "Delete this Node". If you have enabled *Classic Controls*, you need to select (*SHIFT + Left Click*) the node first and then delete (*DEL*) it.
```